rust/noosphere-core/src/data/body_chunk.rs

```diff
@@ -21,7 +27,7 @@ pub struct BodyChunkIpld {
 }
 
 impl BodyChunkIpld {
-    // TODO(#498): Re-write to address potentially unbounded memory overhead
+    #[deprecated]
```

Comment:
Add deprecated note for both of these pointing to preferred alternative https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute